profiling_router.go

```diff
@@ -0,0 +1,147 @@
+package handlers
+
+import (
+	"context"
+	"net/http"
+	"net/http/pprof"
+	"os"
+	"strconv"
+
+	"gopkg.in/errgo.v1"
+
+	"github.com/Scalingo/go-utils/logger"
+)
+
+const PprofRoutePrefix = "/debug/pprof"
```

Comment:
question: does it really need to be public? 🤔 
```suggestion
const pprofRoutePrefix = "/debug/pprof"
```

Reply:
Yes, we need it public for the case we want to add the profiling router with an already existing one like this :
```go
pprofRouter, err := profiling.StartProfilingRoutes(ctx)
if err != nil {
	log.WithError(err).Error("Fail to start profiling routes")
}

globalRouter := mux.NewRouter()
globalRouter.Handle(handlers.PprofRoutePrefix+"/{prop:.*}", pprofRouter) // here
globalRouter.Handle("/{any:.+}", router)
```

Reply:
Oh yes sorry I realize I already asked that 🤦🏾